src/ol_infrastructure/applications/ovs/Pulumi.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: ol-infrastructure-ovs-env
```

Comment:
I wonder if we should expand `ovs` -> `odl-video-service` for sake of clarity? It's easy to get confused about what the acronym is supposed to represent for anyone who isn't familiar with this app.